### DISCUSSION TITLE
Issue with showing stats ,

### DISCUSSION THREAD

**Discussion by kali-37** [anuraghazra/github-readme-stats#3147] [Q&A] [answered]
### Describe the bug

[](https://github.com/subash1237)

### Expected behavior

Expected to show stat as it used to , Recently it's not showing . 

### Screenshots / Live demo link

![2023-08-25_21-21-25](https://github.com/anuraghazra/github-readme-stats/assets/77325527/ff43789e-bba1-4c5e-a579-7906590bd0ee)

**Comment by qwerty541** [comment, marked as answer]
@subash1237 this is because our public instance is being rate-limited by underlying GitHub API (see https://github.com/anuraghazra/github-readme-stats/issues/1471). You can [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) to fix this or wait until the public instance is longer rate-limited.

**Reply by kali-37** [reply]
Thanks @qwerty541 I will try someday  deploy my  my own instance to reduces loads on the vercel instance.
